app/routes.py

```diff
@@ -116,12 +117,12 @@ def verify_pains():
                 found = True
 
             if(not(found)):
-                print('Could not cluster {smile}'.format(smile=smile))
-
-
-
-
-  return render_template('pains_verify_and_coefficient_use.html', title='Cheminformatic Analysis', bad_smiles=bad_smiles, num_remaining=session["num_remaining"], num_removed=session["num_removed"], highest_val=session["highest_val"], lowest_val=session["lowest_val"], reasons_for_failure=reasons_for_failure, include_property=session['include_property'])
+              print('Could not cluster {smile}'.format(smile=smile))
+              cluster_failures.append(smile)
```

Comment:
Nice job, we just need to replicate this change in other try catch inside of `verify_pains`